gtest/src/manager.rs

```diff
@@ -645,9 +674,7 @@ impl JournalHandler for ExtManager {
                 *balance -= value;
             };
 
-            if let Some((_, balance)) = self.actors.get_mut(&to) {
-                *balance += value;
-            };
+            self.add_value_to(to, value);
```

Comment:
I would provide `move_value(from, to, amount)` to cover all value sending needs

Reply:
`add_value/add_value_to` is about minting. Let's rename them. We can provide an additional function to send value but I don't see the use case for it.